docs/src/docs/asciidoc/user/107.adoc

```diff
@@ -1,9 +1,9 @@
 ---
 ---
 = The Ehcache 3.x JSR-107 Provider
-ifndef::sourcedir[]
+ifndef::sourcedir31[]
```

Comment:
If we're going C style on the includes shouldn't this be `ifndef::common_adoc[]`?